src/app/shared/components/addscenario/addscenario.component.ts

```diff
@@ -384,6 +403,9 @@ export class AddScenarioModal implements OnInit, OnDestroy {
         this._settingsService.postEntity(scen, this.configSettings.scenariosURL + '?statisticgroupIDorCode=' + scen.statisticGroupID)
             .subscribe((response: any) => {
                 this._nssService.setSelectedRegion(this.selectedRegion);
```

Comment:
I noticed that when you add/clone a scenario, for a second it shows the entire scenario, then hits the filters. Since the region is already selected, you should be able to remove this line and it should re-query properly with your `requeryFilters` function.